### ISSUE TITLE
[Accessibit Audit] Forms should have better state indicators

### ISSUE THREAD

**Issue by SaptakS** [freedomofpress/pressfreedomtracker.us#2004]
The form elements in https://pressfreedomtracker.us/submit-incident/ doesn't really have proper state indicator for example for active state. The active state, focus state, etc. should maybe have a border to denote the current state, that meets the focus indicator requirements.
